### ISSUE TITLE
Error in patch commands

### ISSUE THREAD

**Issue by jromandigitalprojex** [mrgnhnt96/pub_version_plus#2]
When I try to increase the patch version, it doesn't reset the build version. 

```
$ flutter pub run pub_version_plus:main patch
[station_screen] "1.1.5" => "1.1.6"
$ flutter pub run pub_version_plus:main patch
[station_screen] "1.1.6" => "1.1.7"
$ flutter pub run pub_version_plus:main build
[station_screen] "1.1.7" => "1.1.7+1"
$ flutter pub run pub_version_plus:main build
[station_screen] "1.1.7+1" => "1.1.7+2"
$ flutter pub run pub_version_plus:main patch
[station_screen] "1.1.7+2" => "1.1.8+2"
```

**Comment by jromandigitalprojex** [comment]
This error has been corrected in this pull request that is waiting to be approved. [fixed error reset build counter when incrementing a patch](https://github.com/mrgnhnt96/pub_version_plus/pull/3)

**Comment by mrgnhnt96** [comment]
Thanks for contributing!
Closed via #3
